src/@types/requests.ts

```diff
@@ -107,6 +107,9 @@ export type SendActionDelayedEventRequestOpts = ParentDelayId;
 
 export type SendDelayedEventRequestOpts = SendTimeoutDelayedEventRequestOpts | SendActionDelayedEventRequestOpts;
 
+export function isSendDelayedEventRequestOpts(opts: object): opts is SendDelayedEventRequestOpts {
+    return (opts as TimeoutDelay).delay !== undefined || (opts as ParentDelayId).parent_delay_id !== undefined;
+}
```

Comment:
Nitpick, but this should probably verify that the types of `event_id`, `delay`, `parent_delay_id` match in case anyone were to use `isSendDelayedEventRequestOpts` on truly untrusted input.